src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -30,7 +30,7 @@ let dropdownIndex = 0;
 
 // TODO: ADS-382 allow implementer to add refs to dropdown children. Currently they are being removed in the clone process.
 
-const Dropdown = (props: DropdownProps) => {
+const DropdownMenu = (props: DropdownMenuProps) => {
```

Comment:
thoughts on changing isDropdownOpen to isMenuExpanded to mirror primary nav prop names and aria states?